docs/tutorials/quickstart.md

```diff
@@ -1,186 +1,240 @@
+You can also choose to install aks-engine using [gofish][gofish-about]. To do so, execute the command `gofish install aks-engine`. You can install gofish following the [instructions][gofish-install] for your OS.
 
-If you would prefer to build `aks-engine` from source or you are interested in contributing to `aks-engine` see [building from source](#build-aks-engine-from-source) below.
+If you would prefer to build AKS Engine from source, or you are interested in contributing to AKS Engine, see [the developer guide][developer-guide] for more information.
 
 ## Completion
 
-`aks-engine` supports bash completion. To enable this, add the following to your `.bashrc` or `~/.profile`
+AKS Engine supports bash completion. To enable this, add the following to your `.bashrc` or `~/.profile`
 
 ```bash
 source <(aks-engine completion)
 ```
 
-## Usage
+## Deploy your First Cluster
 
-`aks-engine` reads a JSON [cluster definition](../topics/clusterdefinitions.md) and generates a number of files that may be submitted to Azure Resource Manager (ARM). The generated files include:
+`aks-engine` reads a cluster definition which describes the size, shape, and configuration of your cluster. This guide takes the default configuration of one master and two Linux agents. If you would like to change the configuration, edit `examples/kubernetes.json` before continuing.
 
-1. **apimodel.json**: is an expanded version of the cluster definition provided to the generate command. All default or computed values will be expanded during the generate phase.
-2. **azuredeploy.json**: represents a complete description of all Azure resources required to fulfill the cluster definition from `apimodel.json`.
-3. **azuredeploy.parameters.json**: the parameters file holds a series of custom variables which are used in various locations throughout `azuredeploy.json`.
-4. **certificate and access config files**: orchestrators like Kubernetes require certificates and additional configuration files (e.g. Kubernetes apiserver certificates and kubeconfig).
+The `aks-engine deploy` command automates creation of a Service Principal, Resource Group and SSH key for your cluster. If operators need more control or are interested in the individual steps see the ["Long Way" section below](#aks-engine-the-long-way).
 
-### Generate Templates
+**NOTE:** AKS Engine creates a _cluster_; it _doesn't_ create an Azure Kubernetes Service resource. Clusters that you create using the `aks-engine` command (or ARM templates generated by the `aks-engine` command) won't show up as AKS resources, for example when you run `az aks list`.  Think of `aks-engine` as the, er, engine which AKS uses to create clusters: you can use the same engine yourself, but AKS won't know about the results.
```

Comment:
```suggestion
**NOTE:** AKS Engine creates a _cluster_; it _doesn't_ create an Azure Kubernetes Service(AKS) resource. Clusters that you create using the `aks-engine` command (or ARM templates generated by the `aks-engine` command) won't show up as AKS resources, for example when you run `az aks list`.  Think of `aks-engine` as the, er, engine which AKS uses to create clusters: you can use the same engine yourself, but AKS won't know about the results.
```

Reply:
This is a great note. Thanks for adding this.

Reply:
I'd add a space before the `(AKS)`

Reply:
Thanks!

Reply:
nice use of the suggested change feature :)

Reply:
done!